runtime/src/main/scala/akka/grpc/GrpcServiceException.scala

```diff
@@ -25,12 +56,6 @@ class GrpcServiceException(val status: Status, val metadata: Metadata) extends R
     this(status, metadata.asScala)
   }
 
-  /**
-   * Java API: The response status.
-   */
-  def getStatus: Status =
```

Comment:
This is now inherited from `StatusRuntimeException `

Reply:
That's source compatible but not binary compatibly - but only users that called 'getStatus' would be impacted by that, and this class is marked `ApiMayChange`, so that indeed looks OK to me.